### PR TITLE
gh-2771 Backup flow after key added screen

### PR DESCRIPTION
**Pull request by elgatovital**
Handles #2771

Changes proposed in this pull request:
- Start backup flow after push notification confirmation / skipping when generating keys

### CODE REVIEW

File: Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/GenerateKeyFlow.swift [hunk 1]
```diff
@@ -139,6 +139,10 @@ class GenerateKeyFlow: AddKeyFlow {
         }
         show(keyVC)
     }
+
+    override func didKeyAdded() {
```
Comment (DmitryBespalov) [inline Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/GenerateKeyFlow.swift]: @elgatovital it should also work for Import Key flow

File: Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/AddKeyFlow.swift [hunk 2]
```diff
@@ -123,12 +123,16 @@ class AddKeyFlow: UIFlow {
...
         assert(keyName != nil)
         let vc = factory.keyAdded(address: keyInfo!.address, name: keyName!, keyType: keyType) { [unowned self] in
-            stop(success: true)
+            didKeyAdded()
         }
 
         show(vc)
     }
 
+    func didKeyAdded() {
```
Comment (DmitryBespalov) [inline Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/AddKeyFlow.swift]: This is grammatically incorrect, should be `didAddKey` (:+1:) or `keyAdded`

Comment (elgatovital) [reply Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/AddKeyFlow.swift]: renamed to didDelegateKeySetup()

File: Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/GenerateKeyFlow.swift (hunk 1 above)
Comment (elgatovital) [reply Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/GenerateKeyFlow.swift]: I don't think backup should be started for imported keys (they are already persisted somewhere outside). Everything else works as before

Comment (DmitryBespalov) [reply Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/GenerateKeyFlow.swift]: oh, that's a backup! True, I mis-recognized it as a 'delegate key'.

**Review by DmitryBespalov — APPROVED** [review]

**Pull request merged**